api/profile/profileRouter.js

```diff
@@ -14,9 +14,11 @@ const { validateUser } = require('../middleware/validationMiddleware');
 const dsService = require('../dsService/dsModel');
 validateUser;
 
+//TO-DO: Implement Auth0 using the correct middleware(auth0middleware)
+
 // gets current user profile
 
-router.get('/current_user_profile/', authRequired, async (req, res, next) => {
+router.get('/current_user_profile', authRequired, async (req, res, next) => {
```

Comment:
Good thinking to update this router to properly implement the changes needed for this fix!